splinterd/src/main.rs

```diff
@@ -721,6 +722,9 @@ impl fmt::Display for UserError {
             UserError::TransportError(err) => write!(f, "unable to get transport: {}", err),
             UserError::MissingArgument(msg) => write!(f, "missing required argument: {}", msg),
             UserError::InvalidArgument(msg) => write!(f, "required argument is invalid: {}", msg),
+            UserError::ConfigError(msg) => {
+                write!(f, "Error happened building config object: {}", msg)
```

Comment:
"error occurred" sounds more natural than "error happened", IMO. Also, this should probably start with a lowercase letter to be consistent with the other errors.